src/sql-parser/build.rs

```diff
@@ -64,7 +64,8 @@ fn main() -> Result<()> {
 
         for kw in &keywords {
             buf.writeln(format!(
-                "pub const {}: Keyword = Keyword::{};",
+                "#[allow(dead_code)]
```

Comment:
Sadly, no--the only way (I know of, at least) to introduce keywords if through that file--however, we cannot use the `MAX` const-ified keyword because Rust gets confused trying to resolve it, hence: https://github.com/MaterializeInc/materialize/pull/14317/files#diff-ff4ad701e3b476ee7c4794f262a310147cb7448f5da48c590ea9b5648aa89ff2R2072

If this is too hideous to bear the sight of, glad to pivot to something else (`MAXXX`? `BIGGEST`? :D)

Reply:
Oh, derp--I just needed to qualify `MAX`; fixed